### DISCUSSION TITLE
dynamic url endpoints v0.19.4

### DISCUSSION THREAD

**Discussion by SamuelVirgo** [GrapesJS/grapesjs#4446] [Ideas]
https://github.com/artf/grapesjs/issues/831

In earlier versions of Grapesjs we could do the following:

`editor.StorageManager.get('remote').set({ urlStore: '...', ... })`

It seems this is no longer working on v0.19.4? Is there a new way to accomplish this or can this be added back in?

Note, it does seem to be working like this now:

```
const r = storageManager.getStorageOptions('remote');
r.urlLoad = ...
r.urlStore = ...
```

Unsure if this is now an acceptable method.

**Comment by artf** [comment]
Hi Samuel, yeah correct, now Storages are simple objects containing only the implementation logic, and all the options are passed from the top. 
> Unsure if this is now an acceptable method.

Yes, `getStorageOptions` is used to get the current default options for the storage so, you can use it to update the returned object.

**Reply by SamuelVirgo** [reply]
Thank you for confirming and thanks for the great project!
